### DISCUSSION TITLE
Poseidon Theme Preview broken

### DISCUSSION THREAD

**Discussion by Wiredframe** [primefaces/community#1456] [PrimeNG Templates] [answered]
Hi there, 

the preview of the Poseidon theme is broken, it seems that no styles are being applied: https://poseidon.primeng.org/

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've recently updated the domains, probably there would be a mistake. I've checked from the master branch on poseidon github repo and the latest version in the store, and they work as expected. We'll re-deploy the Poseidon and the issue will be solved in a couple of days.

Thanks for the report!

**Comment by AshishFreak** [comment]
does any have this theme i need it can you please share
